lib/rails_cloudflare_turnstile/engine.rb

```diff
@@ -0,0 +1,9 @@
+module RailsCloudflareTurnstile
+  class Engine < ::Rails::Engine
+    initializer "rails_cloudflare_turnstile.precompile" do |app|
+      %w[javascripts images].each do |sub|
```

Comment:
should we be serving this file ourselves?

Reply:
this is just the mock version I wrote